### PR TITLE
Chore: Change bundle tags color and refactor app details page header styles

### PR DESCRIPTION
**Pull request by rique223**
## Proposed changes (including videos or screenshots)

Increased the right side margin of the app avatar, removed the 'by' substring from the app author metadata, and changed the font scale and colors for the metadata section and short description.
Demo image:
image

### CODE REVIEW

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 1]
```diff
@@ -15,19 +15,28 @@ type BundleChipsProps = {
...
-					variant='primary'
-				>
-					{bundle.bundleName}
-				</Tag>
+			{bundledIn.map((bundle) => (
+				<Fragment key={bundle.bundleId}>
+					<Box ref={bundleRef} onMouseEnter={(): void => setIsHovered(true)} onMouseLeave={(): void => setIsHovered(false)}>
+						<Tag variant='featured'>{bundle.bundleName}</Tag>
+					</Box>
+					<PositionAnimated
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:27]: just passing a title to the component is enough to dispatch a tooltip!

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 1 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx]: Solved

File: apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx [hunk 2]
```diff
@@ -36,16 +49,41 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
...
+					<Box fontScale='c1' color='stroke-light'>
+						|
 					</Box>
+					<Box fontScale='c1' mi='x16'>
+						{t('Version_version', { version })}
+					</Box>
+
+					{lastUpdated && (
+						<>
+							<Box fontScale='c1' color='stroke-light'>
```
Comment (juliajforesti) [inline apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx:76]: The `color` prop is for font color, you shouldn't pass a stroke token to it, only font colors :)

**Review by juliajforesti — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 3]
```diff
@@ -15,19 +15,22 @@ type BundleChipsProps = {
...
-					title={t('this_app_is_included_with_subscription', {
-						bundleName: bundle.bundleName,
-					})}
-					variant='primary'
-				>
-					{bundle.bundleName}
-				</Tag>
+			{bundledIn.map((bundle) => (
+				<Fragment key={bundle.bundleId}>
+					<Box ref={bundleRef}>
```
Comment (juliajforesti) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:23]: I didn't get the point of this ref 🥲
Is it doing something?

Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:23]: Not right now, in the past it connected a tooltip to the bundle tag. I'll fix this.

File: apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx (hunk 2 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx]: Solved

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 3 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:23]: Solved

**Review by juliajforesti — APPROVED** [review]

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 4]
```diff
@@ -18,16 +18,19 @@ const BundleChips = ({ bundledIn }: BundleChipsProps): ReactElement => {
...
-					key={index}
-					title={t('this_app_is_included_with_subscription', {
-						bundleName: bundle.bundleName,
-					})}
-					variant='primary'
-				>
-					{bundle.bundleName}
-				</Tag>
+			{bundledIn.map((bundle) => (
+				<Fragment key={bundle.bundleId}>
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:22]:
```suggestion
				
```

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 5]
```diff
@@ -18,16 +18,19 @@ const BundleChips = ({ bundledIn }: BundleChipsProps): ReactElement => {
...
+					<Box>
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:23]:
```suggestion

```

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 6]
```diff
@@ -18,16 +18,19 @@ const BundleChips = ({ bundledIn }: BundleChipsProps): ReactElement => {
...
+						<Tag
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:24]:
```suggestion
						<Tag key={bundle.bundleId}
```

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 7]
```diff
@@ -1,7 +1,7 @@
-import { Tag } from '@rocket.chat/fuselage';
+import { Box, Tag } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import type { ReactElement } from 'react';
-import React from 'react';
+import React, { Fragment } from 'react';
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:4]:
```suggestion
import React from 'react';
```

File: apps/meteor/client/views/admin/apps/BundleChips.tsx [hunk 8]
```diff
@@ -1,7 +1,7 @@
-import { Tag } from '@rocket.chat/fuselage';
+import { Box, Tag } from '@rocket.chat/fuselage';
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/BundleChips.tsx:1]:
```suggestion
import { Tag } from '@rocket.chat/fuselage';
```

File: apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx [hunk 9]
```diff
@@ -36,24 +49,45 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
 					</Box>
 					{bundledIn && Boolean(bundledIn.length) && <BundleChips bundledIn={bundledIn} />}
 				</Box>
-				{app?.shortDescription && <Box mbe='x16'>{app.shortDescription}</Box>}
+
+				{shortDescription && (
+					<Box fontScale='p1' color='default' mbe='x16'>
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx:54]:
```suggestion
					<Box fontScale='p1' mbe='x16'>
```

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 4 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:22]: Solved

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 5 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:23]: Solved

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 6 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:24]: Solved

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 7 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:4]: Solved

File: apps/meteor/client/views/admin/apps/BundleChips.tsx (hunk 8 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/BundleChips.tsx:1]: Solved

File: apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx (hunk 9 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx:54]: Solved

File: apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx [hunk 10]
```diff
@@ -36,24 +49,45 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
...
-				<Box display='flex' flexDirection='row' color='hint' alignItems='center'>
-					<Box fontScale='p2m' mie='x16'>
-						{t('By_author', { author: author?.name })}
+				<Box fontScale='c1' display='flex' flexDirection='row' color='hint' alignItems='center'>
+					{author?.name}
+					<Box mi='x16' color='disabled'>
+						|
 					</Box>
-					<Box is='span'> | </Box>
+					{t('Version_version', { version })}
```
Comment (dougfabris) [inline apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx:68]: `version` are duplicated

**Review by dougfabris — APPROVED** [review]

**Pull request merged**